esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2227,19 +2227,19 @@ void CFileSprayEx::getDropZoneInfoByIP(double clientVersion, const char* ip, con
         throw MakeStringExceptionDirect(ECLWATCH_INVALID_IP, "Network address must be specified for a dropzone!");
 
     Owned<IEnvironmentFactory> factory = getEnvironmentFactory();
-    Owned<IConstEnvironment> env = factory->openEnvironment();
-    if (!env)
+    Owned<IConstEnvironment> constEnv = factory->openEnvironment();
+    if (!constEnv)
         throw MakeStringException(ECLWATCH_CANNOT_GET_ENV_INFO, "Failed to get environment information.");
 
-    SCMStringBuffer directory;
     StringBuffer destFile;
     if (isAbsolutePath(destFileIn))
     {
         destFile.set(destFileIn);
-        Owned<IConstDropZoneInfo> dropZone = env->getDropZoneByAddressPath(ip, destFile.str());
+        Owned<IConstDropZoneInfo> dropZone = constEnv->getDropZoneByAddressPath(ip, destFile.str());
         if (!dropZone)
             throw MakeStringException(ECLWATCH_DROP_ZONE_NOT_FOUND, "Dropzone not found for network address %s.", ip);
 
+        SCMStringBuffer directory;
         dropZone->getDirectory(directory);
         destFileOut.set(destFile.str());
         if ((destFile.length() >= directory.length()) && !strnicmp(destFile.str(), directory.str(), directory.length()))
```

Comment:
You shouldn't need this..
getDropZoneByAddressPath ensures this returns a dropzone that matches destfile already